src/DataFrames.jl

```diff
@@ -128,4 +128,18 @@ for (dir, filename) in [
     include(joinpath(dir, filename))
 end
 
+# Import  "StringEncodings" if available and set up decoder
+_decoder=nothing
+ENCODING_PACKAGE_READY = false
+function __init__()
+    try Pkg.installed("StringEncodings")
```

Comment:
@ararslan How do you feel about adding a dependency on StringEncodings? It shouldn't be an issue since the package is very small, and it only relies on iconv which is already installed on all major platforms. But I'm biased so I'll let you decide.